pkgs/applications/video/screenkey/default.nix

```diff
@@ -0,0 +1,42 @@
+{ lib
+, buildPythonApplication
+, fetchFromGitHub
+, distutils_extra
+, setuptools-git
+, intltool
+, pygtk
+, libX11
+, libXtst
+}:
+buildPythonApplication rec {
+  name = "screenkey-0.9";
+
+  src = fetchFromGitHub {
```

Comment:
Why don't you use https://www.thregr.org/~wavexx/software/screenkey/releases/screenkey-0.9.tar.gz?
Also, you could specify `pname` and `version` and leave out `name`.
Sorry for the many comments :)

Reply:
IDK, `fetchFromGitHub` is more convenient to use. GitHub is also less likely to die or move.

Don't see a point in using `pname` and `version`—I am using `name` as `rev` (and reconstructing it from `pname` and `version` would remove all the benefit)

Reply:
Alright, as you wish.

Reply:
Please do use `pname` and `version` since in long term `buildPython*` will likely get rid of `name`.

Reply:
ok, fixed